docs/cli.md

```diff
@@ -1071,7 +1074,7 @@ viam packages export --org-id=<org-id> --name=<package-name> --version=latest --
 | -------- | ----------- | ------------------- | -------- |
 | `--org-id` | The organization ID of the package | `upload`, `export` | **Required** |
 | `--name` | The name of the package | `upload`, `export` | **Required** |
-| `--version` | The version of the package or `latest` | `upload`, `export` | **Required** |
+| `--version` | The version of the package | `upload`, `export` | **Required** |
```

Comment:
I think training scripts aren't versioned and I don't actually know where versions for ML models etc are currently displayed. So I think having that is quite important.
```suggestion
| `--version` | The version of the package or `latest` | `upload`, `export` | **Required** |
```

Reply:
Ok interesting. Whatever I was doing on the train I concluded that latest wasn't working/accepted but I must have been confused